src/api/app/models/bs_request.rb

```diff
@@ -92,7 +92,9 @@ class SaveError < APIError
   has_many :bs_request_actions, dependent: :destroy
   has_many :reviews, dependent: :delete_all
   has_many :comments, as: :commentable, dependent: :delete_all
-  has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
+  has_many :request_history_elements, -> { order(:created_at) },
```

Comment:
Is this change needed? It seems to be the same code, just split up on multiple lines.

Reply:
No, it's not really needed. Reverting